recipe/meta.yaml

```diff
@@ -108,41 +99,32 @@ outputs:
       build:
         - {{ compiler('cxx') }}
         - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
-        - autoconf                # [not win]
-        - automake                # [not win]
-        - libtool                 # [not win]
         - cmake
         - libgomp                 # [linux]
-        - llvm-openmp             # [osx or win]
-        - autotools_clang_conda   # [win]
+        - llvm-openmp             # [osx]
```

Comment:
`llvm-openmp` doesn't seem to be necessary on windows after all (even testing [against](https://github.com/conda-forge/faiss-split-feedstock/pull/17#issuecomment-741944363) other blas versions)